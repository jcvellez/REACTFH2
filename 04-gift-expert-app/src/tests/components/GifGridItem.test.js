//import { shallow } from "enzyme"
import React from 'react'
import { shallowToJson } from "enzyme-to-json"
import GifGidItem from "../../GifGridItem"

describe('Pruebas en <GifGridItem />', () => { 

    const name = 'Un titulo';
    const url = 'http:/localhost/algo.jpg'; 

    test('debe de mostrar el componente correctamente', () => {
      const wraper =  shallowToJson( <GifGidItem name={ name } url={ url } /> )
      expect( wraper ).toMatchSnapshot();
    })    
 })