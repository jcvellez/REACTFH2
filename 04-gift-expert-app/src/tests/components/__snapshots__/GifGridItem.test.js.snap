// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifGridItem /> debe de mostrar el componente correctamente 1`] = `null`;
